lib/cloudfront/manager/Manager.rb

```diff
@@ -112,6 +112,28 @@ def create(local)
         puts "Failed to create distribution #{local.file_name}\n#{e}"
       end
 
+      def invalidate(invalidation_name)
+
+        invalidation = Loader.invalidation(invalidation_name)
+
+        # Use a combination of the current time and md5 of paths to prevent
+        # identical invalidations from being ran too often
+        time_throttle = (Time.now.to_i / 60 / 5)
+        md5 = Digest::MD5.hexdigest(invalidation.paths.join)[0..5]
+
+        @cloudfront.create_invalidation({
+          distribution_id: invalidation.distribution_id,
+          invalidation_batch: {
+            paths: {
+              quantity: invalidation.paths.size,
+              items: if invalidation.paths.empty? then nil else invalidation.paths end
```

Comment:
Just discovered that this can actually be written as:

```ruby
items: if !invalidation.paths.empty? then invalidation.paths end
```

In the case that `invalidation.paths` is empty, the nil will be supplied without the else branch.